common/src/main/scala/com/snowplowanalytics/snowplow/storage/bigquery/common/LoaderRow.scala

```diff
@@ -167,9 +176,9 @@ object LoaderRow {
 
   implicit val bqModeEncoder: Encoder[Mode] = deriveEncoder[Mode]
 
-  implicit def bqTypeEncoder: Encoder[Type] = deriveEncoder[Type]
```

Comment:
Wondering why I wrote it this way in a first place.

Reply:
What other way would be better? Manual derivation? Full auto?

Reply:
Ah, nvm, saw what you mean.